src/index.ts

```diff
@@ -380,6 +327,42 @@ export class GalleryComponent extends BaseComponent {
     return null;
   }
 
+  private _galleryThumbsTemplate = (thumb): string => `

```

Comment:
no `Thumb` type for _galleryThumbsTemplate thumb arg due 
to missing id field on Thumbs interface in manifesto package